File: lib/src/main/java/uk/iatom/rfm69/registers/TestLna.java
package uk.iatom.rfm69.registers;

public enum TestLna {
       // RegTestLna),
       NORMAL(0x1B),
       HIGH_SENSITIVITY(0x2D);

       private byte val;
       
       TestLna(int val) {
           this.val = (byte) val;
       }
   
       public byte getVal() {
           return val;
       }
}

File: lib/src/main/java/uk/iatom/rfm69/register/values/TestLna.kt
package uk.iatom.rfm69.register.values

import uk.iatom.rfm69.register.IRegisterValue

enum class TestLna(`val`: Int): IRegisterValue { // RegTestLna),
    NORMAL(0x1B),
    HIGH_SENSITIVITY(0x2D);

    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
